status-im/status-react
# feat: add create profile to onboarding

fixes: https://github.com/status-im/status-mobile/issues/15022

Allows user to set their custom color, user name and their profile picture.

For now it is using the old profile picture code but this will be updated in future work to a new design.

Additionally on ios there is a flickering issue which happens when the image is set:
https://github.com/ivpusic/react-native-image-crop-picker/issues/1658

To fix this we need to upgrade the underlying dependency and will get done outside of this pr.

**TO TEST**
Go to the new Generate Keys flow.
You should be then able to set your profile picture, user name and custom color and continue navigation until you generate keys and then land on the home page of the app.

Should be able to create a profile with or without a profile picture
